core/src/main/scala/org/dbpedia/spotlight/util/MathUtil.scala

```diff
@@ -1,5 +1,7 @@
 package org.dbpedia.spotlight.util
 
+import breeze.linalg.{DenseVector, Transpose}
+import breeze.numerics._
```

Comment:
I think it would be better to import the specific classes that you use here.
Also add an extra line at the end of the file to remove the weird EOF.